packages/tools/examples/viewReferencePresentation/index.ts

```diff
@@ -0,0 +1,517 @@
+const toolGroupId = 'MY_TOOLGROUP_ID';
+const viewportId0 = 'CT_STACK';
+const viewportId1 = 'CT_AXIAL';
+const viewportId2 = 'CT_SAGITTAL';
+const viewportId3 = 'CT_CORONAL';
+const viewportId4 = 'TEST_STACK';
+const viewportIds = [
+  viewportId0,
+  viewportId1,
+  viewportId2,
+  viewportId3,
+  viewportId4,
+];
+let viewport;
+const viewports = [];
+const renderingEngineId = 'myRenderingEngine';
+
+// ======== Set up page ======== //
+setTitleAndDescription(
+  'ViewReferencePresentation',
+  'Here we demonstrate view reference/presentation syncing.'
+);
+
+const width = `18vw`;
+const height = `33vh`;
+const content = document.getElementById('content');
+const viewportGrid = document.createElement('div');
+
+viewportGrid.style.width = '100%';
+
+const element0 = document.createElement('div');
+const element1 = document.createElement('div');
+const element2 = document.createElement('div');
+const element3 = document.createElement('div');
+const element4 = document.createElement('div');
+const elements = [element0, element1, element2, element3, element4];
+const controlElements = [...elements];
+const setViewElements = controlElements.map(() => {
+  const newElement = document.createElement('div');
+  elements.push(newElement);
+  return newElement;
+});
+
+elements.forEach((element) => {
+  Object.assign(element.style, {
+    width,
+    height,
+    background: 'red',
+    display: 'inline-block',
+  });
+  // Disable right click context menu so we can have right click tools
+  element.oncontextmenu = (e) => e.preventDefault();
+  viewportGrid.appendChild(element);
+  element.onclick = (evt) => {
+    const clickViewport = getEnabledElement(element)?.viewport;
+    viewport = clickViewport;
+    return false;
+  };
+});
+
+content.appendChild(viewportGrid);
+
+const instructions = document.createElement('p');
+instructions.innerText = `
+  There are two lines of viewports, the top line allows controls, while the bottom line shows what happens when a isReferenceViewable applies, and the
+  setView is updated when it is viewable.
+  The left and right most views are stack views, while the middle three are  orthographic views of the left most viewport.
+  Turning on apply orientation will apply the orientation from the source to the destination viewports for compatible viewports.
+  Turning on apply presentation will apply the presentation for viewports that are not compatible with the view reference.
+  There is no automatic conversion of stack to volume to demonstrate compatibility for conversion of viewport types.
+  Note the two acquisition viewports (left two viewports) - these will sync in opposite scroll directions for the destination viewports.
+  `;
+
+content.append(instructions);
+
+const leftTopDisplayArea = {
+  storeAsInitialCamera: true,
+  imageArea: [1.1, 1.1],
+  imageCanvasPoint: {
+    imagePoint: [0, 0],
+    canvasPoint: [0, 0],
+  },
+};
```

Comment:
I noticed we lack documentation for `displayArea` here. Could you please add it? Specifically for `storeAsInitialCamera`, please provide at least a brief explanation about its behavior when set to `false` or `true` (not an entire paragraph, but a concise description).

packages/core/src/types/displayArea.ts

```ts
type DisplayArea = {
  type?: 'SCALE' | 'FIT';
  scale?: number;
  interpolationType?: InterpolationType;
  imageArea?: [number, number]; // areaX, areaY
  imageCanvasPoint?: {
    imagePoint: [number, number]; // imageX, imageY
    canvasPoint?: [number, number]; // canvasX, canvasY
  };
  storeAsInitialCamera?: boolean;
};
```